evaluations.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Evaluations-MockUp</title>
+		<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+		<link rel="stylesheet" type="text/css" href="css/login.css">
+		<link rel="stylesheet" type="text/css" href="css/evaluations.css">
+	</head>
+	<body>
+	<!-- Header --> 
+		<header>
+			<nav>
+				<div id="logo" class="logoClass "></div>
+				<div id="navigation" class="navigationClass">
+					<div id="button1" class="buttoni button1 formated selected">
+					Evaluations
+					</div>
+					<a href="new.html"><div id="button2" class="buttoni button2 formated">
```

Comment:
I've created this divs to be buttons. For design purposes. After that I wanted to switch pages more easily by pressing those buttons. So one solution is to wrap it with an "a". I was not thinking about the whole thing in the beginning.